nixos/tests/sway.nix

```diff
@@ -121,8 +123,15 @@ import ./make-test-python.nix ({ pkgs, lib, ... }:
     machine.wait_for_text("You pressed the exit shortcut.")
     machine.screenshot("sway_exit")
 
+    swaymsg("exec swaylock")
+    machine.wait_until_succeeds("pgrep -x swaylock")
+    machine.sleep(3)
+    machine.send_chars("${nodes.machine.config.users.users.alice.password}")
+    machine.send_key("ret")
+    machine.wait_until_fails("pgrep -x swaylock")
```

Comment:
Does this mean we need to wait 15 minutes (the default timeout) for this to fail?

Reply:
Yes, IIRC the default timeout was something very long like that (not sure why though...).